tests/tasks/cleanup_candlepin_container.yml

```diff
@@ -0,0 +1,37 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Check if the candlepin container is started
+  command: podman container exists candlepin
+  register: __rhc_candlepin_cont_started
+  failed_when: false
+  changed_when: false
+
+- name: Stop and potentionally remove Candlepin container
+  command: podman stop candlepin
+  when: __rhc_candlepin_cont_started.rc == 0
+  changed_when: true
+
+- name: Check if there is a stopped candlepin container
+  command: podman ps -a --filter name=candlepin
+  register: __rhc_candlepin_container_stopped
+  failed_when: false
+  changed_when: false
+
+- name: Remove Candlepin container
+  command: podman rm candlepin
+  changed_when: true
+  when:
+    - '"candlepin" in __rhc_candlepin_container_stopped.stdout'
+
+- name: Remove Candlepin container image
```

Comment:
@richm maybe let's not clean up the image? It won't affect anything, and re-downloading it with each test upstream is a waste of network bandwidth.

Reply:
> @richm maybe let's not clean up the image? It won't affect anything, and re-downloading it with each test upstream is a waste of network bandwidth.

Ok.  That will affect the podman tests, in the scenarios where we run tests from multiple roles - the podman tests expect no images.  I guess that means changing this PR to use `__rhc_cleanup_image: false` (or just omit __rhc_cleanup_image because the default is false)

Reply:
I'll fix the podman tests to work if there are images on the system.